code/modules/mob/living/simple_animal/friendly/lizard.dm

```diff
@@ -18,3 +18,61 @@
 	mob_property_flags = MOB_NO_PETRIFY //Can't get petrified (nethack references)
 	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/animal/lizard
 	held_items = list()
+
+	stop_automated_movement_when_pulled = TRUE
+	environment_smash_flags = FALSE
```

Comment:
not a fan of using FALSE here, it's a bitflag, I'd use 0 if there's no more suitable define